package-lock.json

```diff
@@ -1,8 +1,8147 @@
 {
   "name": "azure-rest-api-specs-tests",
   "version": "0.1.0",
-  "lockfileVersion": 1,
```

Comment:
could you revert this change, as it's unrelated

Reply:
Will do in the next update